YcmdCompletion.sublime-settings

```diff
@@ -46,4 +46,19 @@
     and other tools, uncomment this:
   */
   // "python_binary_path": "/usr/local/bin/python"
+
+  /*
+    The languages you wish to use ycmd for.
+    Supported: cpp, rust
+
+    The following languages may also work, but are untested with this plugin:
+    c, erlang, java, d, lua, python,
+    ocaml, js, perl, go, objc,
+    cpp, cs, php, ruby, rust
+
+    These languages may also work if the corresponding Sublime
+    syntax plugin is installed:
+    vb, typescript, elixir
+  */
```

Comment:
So there's basically no reason these other languages shouldn't work, which is why I don't want to remove them for the list entirely :smile: